DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/classes/DSACampaign.java

```diff
@@ -14,38 +14,69 @@
 
 package com.google.sps.classes;
 
-// contains all the necessary information about a DSA campaign, including id's, input settings, and statistics
+// contains all the necessary information about a DSA campaign
 public class DSACampaign {
 
-	public int DSACampaignId;
-    public int userId;
-    public int keywordCampaignId;
+    // unique identifier for the DSA campaign
+	public String DSACampaignId;
+    // id of the user who created the DSA campaign
+    public String userId;
+    // id of the keyword campaign this DSA campaign was run on
+    public String keywordCampaignId;
 
     // settings
+
+    // name of the DSA campaign (used for display purposes)
 	public String name;
-    public String fromDate;
-    public String toDate;
+    // status of the blackbox data retrieval: pending (stats still pending) or complete (stats retrieved)
+    public String campaignStatus;
+    // start date of the DSA campaign: MM/DD/YYYY
+    public String startDate;
+    // end date of the DSA campaign: MM/DD/YYYY
+    public String endDate;
+    // how much an advertiser spends each time a user clicks on their ad link
+    public double manualCPC;
+    // amount an advertiser is willing to spend each day
     public double dailyBudget;
-    public String location;
+    /*
+     * space-separated string containing the different locations the advertiser wants to target
```

Comment:
How will space-seperating work for your example "California, United States of America" and other locations?

Reply:
So in that example I was just trying to say that you have to write California and United States of America instead of CA and USA. You wouldn't ever have commas. Space-separating would work like "Texas California Philadelphia". I just realized though that this wouldn't work for locations with multiple words, so I'm going to change it to a comma-separated string.

Reply:
I see, can you add an example of what this field could contain.

Reply:
Added examples and more comments.